components/renku_data_services/session/orm.py

```diff
@@ -139,6 +150,7 @@ def dump(self) -> models.SessionLauncher:
             description=self.description,
             environment_kind=self.environment_kind,
             environment_id=self.environment_id if self.environment_id is not None else None,
+            resource_class_id=self.resource_class_id if self.resource_class_id is not None else None,
```

Comment:
nitpick: I don't think this if actually does something, same for the line above. we can just assign directly.